src/extension/intellisenseHelper.ts

```diff
@@ -182,7 +182,7 @@ export class IntellisenseHelper {
             let setEnvironmentVariableCommand: string = "";
             if (os.type() === "Darwin") {
                 setEnvironmentVariableCommand = "launchctl setenv VSCODE_TSJS 1";
-            } else if (os.type() === "Windows") {
+            } else if (os.type() === "Windows_NT") {
                 setEnvironmentVariableCommand = "setx VSCODE_TSJS 1";
             }
```

Comment:
Should we give an error / warning in the Linux case (or any other case) in an `else` at the end here? Right now it seems like on Linux we are just spawning a child_process with an empty command